simpletransformers/conv_ai/conv_ai_model.py

```diff
@@ -730,6 +730,44 @@ def interact(self, personality=None):
             history = history[-(2 * args["max_history"] + 1) :]
             out_text = tokenizer.decode(out_ids, skip_special_tokens=True)
             print(out_text)
+    
+    def interactSingle(self, question, history=list(), personality=None):
```

Comment:
```suggestion
    def interact_single(self, question, history=list(), personality=None):
```
Do you mind changing the function name to `interact_single`?